RFS/src/main/java/com/rfs/common/SnapshotShardUnpacker.java

```diff
@@ -72,40 +70,10 @@ public Path unpack() {
         }
     }
 
-    @Override
-    public void close() {
```

Comment:
do we care about these things not getting cleaned up?

Reply:
No, we want to recycle tasks between shards with ECS. When this was in there, we were deleting the shard date before we reindexed it